buildSrc/src/main/kotlin/ToolchainForTests.kt

```diff
@@ -9,12 +9,11 @@ val Project.javaToolchains: JavaToolchainService
 
 fun Project.addAdditionalJdkVersionTests() {
     // Tests should be run on all supported LTS versions of the JDK. For example, see https://endoflife.date/java
-    addJdkVersionTests("testOnJdk11", 11)
-    addJdkVersionTests("testOnJdk17", 17)
+    arrayOf(8, 11, 17).forEach(::addJdkVersionTests)
```

Comment:
Besides the LTS release, we also should include the latest release (e.g. 18).

Reply:
Seems tests failed on JDK 18, maybe we should handle it in a new PR?

<img width="764" alt="image" src="https://user-images.githubusercontent.com/10363352/165200289-b8da94ba-113a-49b0-9b6e-80d70c4a1451.png">

Reply:
See https://github.com/Goooler/ktlint/pull/20, something wrong with junit5?

Reply:
Yes, it might be better to resolve in a separate PR, tnx.